index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   
+   <header id="header">
+     <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="" id="header-img">
+
+     <nav id="nav-bar">
+       <a href="#products" class="nav-link" >Elemnet1</a>
+       <a href="#products" class="nav-link">Element2</a>
+       <a href="#products" class="nav-link">Elemnet3</a>
```

Comment:
To follow the semantic HTML tags, It would be nice if your use the <`ul`> tags to make the element look like this <`li`><`a href="#products" class="nav-link`">Element1<`/a></li`>.